lighthouse-core/report/html/renderer/category-renderer.js

```diff
@@ -350,10 +349,33 @@ class CategoryRenderer {
       percentageEl.title = Util.i18n.strings.errorLabel;
     }
 
+    if (this.hasApplicableAudits(category)) {
+      wrapper.classList.add(`lh-gauge__wrapper--${Util.calculateRating(category.score)}`);
+    } else {
+      // render gray n/a for entirely non-applicable categories
```

Comment:
```suggestion
      // Render gray n/a for entirely non-applicable categories.
```

nit: we capitalize and end comments with periods.